int8 vehicle_ID
bool started_ack